src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -302,6 +302,10 @@ export default class ScopeModel {
             return [false, 'ERR: SCRATCHPAD MAX 3 CHAR'];
         }
 
+        if (scratchPadText === '.') {
+            return radarTargetModel.setDefaultScratchpad(radarTargetModel.aircraftModel);
+        }
```

Comment:
Don't need to pass anything here, since the RadarTargetModel can access its own aircraftModel.